src/src/DataStructs/SettingsStruct.h

```diff
@@ -194,6 +194,9 @@ class SettingsStruct_tmpl
   byte          ETH_Subnet[4];
   byte          ETH_DNS[4];
   uint8_t       ETH_Wifi_Mode;
+  int8_t        I2C_Multiplexer_Addr;
+  int8_t        I2C_Multiplexer_Port[N_TASKS];
+  int8_t        I2C_Multiplexer_Type;
```

Comment:
Better have the type before the port values.
This way we have the nr task related options at the end (for as long as it may last)

Reply:
The Type field was an afterthought, when I already ran it on both an ESP32 and ESP8266. I'll move it around, and clear/fix my local settings, of course.